db/re/item_db.txt

```diff
@@ -3119,6 +3119,25 @@
 4948,Skill_Delay1,After Skill Delay Lv1,6,10,,,,,,,,,,,,,,,{ bonus bDelayrate,-2; },{},{}
 4949,Skill_Delay2,After Skill Delay Lv2,6,10,,,,,,,,,,,,,,,{ bonus bDelayrate,-4; },{},{}
 4950,Skill_Delay3,After Skill Delay Lv3,6,10,,,,,,,,,,,,,,,{ bonus bDelayrate,-6; },{},{}
+4951,Darkness_Drop,Darkness Drop,6,0,,,,,,,,,,,,,,,{ bonus2 bAddEle,Ele_Dark,2; bonus2 bMagicAtkEle,Ele_Dark,2; bonus2 bSubEle,Ele_Dark,1; },{},{}
```

Comment:
I'm sceptical about the magical effect.
- Increases physical damage on dark monster.
- Reduces damage from dark attack.
- **Increases damage of dark element magic**. There is no magic skill element dark (for player) if I'm not wrong. Also the previous effects apply vs dark monsters but this one deal more damage vs monsters of opposite element in the table of element.

The scripts match the description. Wrong description / translation ?

Reply:
There is actually 1 dark magic skill: Hell Inferno.

Reply:
I will merge it like that for now.
We can change it later on anyway. :-)